cmd/neofs-cli/modules/storagegroup/put.go

```diff
@@ -39,7 +40,7 @@ func initSGPutCmd() {
 	flags.String("cid", "", "Container ID")
 	_ = sgPutCmd.MarkFlagRequired("cid")
 
-	flags.StringSliceVarP(&sgMembers, sgMembersFlag, "m", nil, "ID list of storage group members")
+	flags.StringSliceVarP(&sgMembers, sgMembersFlag, "m", nil, "ID list of unique storage group members")
```

Comment:
I think there is no need to explain it here. "members" somewhat implies uniqueness.

Reply:
removed